src/test/java/integrationtest/CompanyRestApiIntegrationTest.java

```diff
@@ -247,4 +247,801 @@ public class CompanyRestApiIntegrationTest {
         """, timeStamp), response, new CustomComparator(JSONCompareMode.STRICT, new Customization("timestamp", (((o1, o2) -> true
         )))));
     }
+
+    @Test
+    @DataSet(value = "datasets/companies.yml")
+    @Transactional
+    void 企業登録ができること() throws Exception {
+        String response = mockMvc.perform(MockMvcRequestBuilders.post("/companies")
+                        .contentType(MediaType.APPLICATION_JSON_VALUE)
+                        .content("""
+                        {
+                            "companyName": "未登録株式会社",
+                            "companyPhone": "0312345678",
+                            "region": "神奈川県",
+                            "city": "川崎市",
+                            "address": "高津区1-1-1",
+                            "companyRank": "S",
+                            "salesPersonId": "1"
+                        }
+                        """))
+                .andExpect(MockMvcResultMatchers.status().isCreated())
+                .andReturn().getResponse().getContentAsString(StandardCharsets.UTF_8);
+
+        JSONAssert.assertEquals("""
+        {
+                "message": "企業が正常に登録されました。",
+                "ID": 5
+        }
+        """, response, new CustomComparator(JSONCompareMode.STRICT,
+                new Customization("ID", (((o1, o2) -> true)))));
```

Comment:
ここってこんなに括弧で囲む必要ってなかったりしますかね？
```suggestion
                new Customization("ID", (o1, o2) -> true)));
```
https://nainaistar.hatenablog.com/entry/2020/06/15/Java%E3%81%A7Json%E3%82%92%E6%AF%94%E8%BC%83%E3%81%99%E3%82%8B%EF%BC%88%E7%89%B9%E5%AE%9A%E3%81%AE%E9%A0%85%E7%9B%AE%E3%82%92%E7%84%A1%E8%A6%96%E3%81%99%E3%82%8B%E3%82%84%E3%82%8A%E6%96%B9%EF%BC%89

Reply:
削除いたしました。
なぜそうなったかわからないです...

→[57b1e0b](https://github.com/ABECKCROW/s-sfa-api/pull/21/commits/57b1e0b4ccf97d3dc8f4f1157f95c4e43fe9ef4d)